{
  "messages": [
    {
      "error_id": "",
      "references": [
        {
          "column": 24,
          "file": "/home/yo30qaqy/data-race-detection-benchmark-suite/results-20250730-145557/SPMDIR-MUST/P2P/LocalConcurrency-receive-mpi_irecv-001.c",
          "line": 37
        },
        {
          "column": 31,
          "file": "/home/yo30qaqy/data-race-detection-benchmark-suite/results-20250730-145557/SPMDIR-MUST/P2P/LocalConcurrency-receive-mpi_irecv-001.c",
          "line": 39
        }
      ],
      "text": "Found recv operation/call at results-20250730-145557/SPMDIR-MUST/P2P/LocalConcurrency-receive-mpi_irecv-001.c:37:24 which is in conflict with store operation/call at results-20250730-145557/SPMDIR-MUST/P2P/LocalConcurrency-receive-mpi_irecv-001.c:39:31 before release",
      "type": "Local Data Race"
    }
  ]
}
